File: FizzBuzzBizz.java
public class FizzBuzzBizz{
	public static void main(String[] args){
	for(int i=1; i<=100; i++){

		if( i % 35 == 0 ){
			System.out.println( "BuzzBizz" );

		}else if( i % 15 == 0 ){
			System.out.println( "FizzBuzz" );

		}else if( i % 7 == 0 ){
			System.out.println( "Bizz" );

		}else if( i % 5 == 0 ){
			System.out.println( "Buzz" );

		}else if( i % 3 == 0 ){
			System.out.println( "Fizz" );

		}else{
			System.out.println( i );

		}

	}

	}
}

File: FizzBuzzBizz.hs
main :: IO ()
main = do
    putStrLn "Enter a number: "
    x <- readLn
    putStrLn ""
    fizzbuzz x

fizzbuzzbizz :: Int -> IO ()
fizzbuzzbizz 0 = putStrLn "0";
fizzbuzzbizz x = do
        fizzbuzzbizz $ x - 1
        case (mod x 7, mod x 5, mod x 3) of
            (0, 0, 0) -> putStrLn $ show x ++ " FizzBuzzBizz"
            (0, 0, _) -> putStrLn $ show x ++ " FizzBuzz"
            (0, _, 0) -> putStrLn $ show x ++ " FizzBizz"
            (0, _, _) -> putStrLn $ show x ++ " Fizz"
            (_, 0, 0) -> putStrLn $ show x ++ " BuzzBizz"
            (_, 0, _) -> putStrLn $ show x ++ " Buzz"
            (_, _, 0) -> putStrLn $ show x ++ " Bizz"
            (_, _, _) -> print x
